Mathlib/Probability/Density.lean

```diff
@@ -411,6 +424,34 @@ end IsUniform
 
 end
 
+section TwoVariables
+
+open ProbabilityTheory
+
+variable {F : Type*} [MeasurableSpace F] {ν : Measure F} {X : Ω → E} {Y : Ω → F}
+variable [HasPDF (fun ω ↦ (X ω, Y ω)) ℙ (μ.prod ν)]
```

Comment:
```suggestion
```
You wrote this one in the theorem statement. No need to have it here as well.

Reply:
Thank you! Sorry about that, I just edited again to delete this line.